HIPAA updates part deux (#7725)
PostHog/posthog.com: contents/docs/privacy/hipaa-compliance.md

PostHog offers a third approach without either of these downsides: hosting the product analytics systems yourself.

## Does PostHog offer a BAA for PostHog Cloud?

> Yes. If you're interested in a BAA for HIPAA compliance, please contact [sales@posthog.com](mailto:sales@posthog.com) with information about your requirements.  

Reviewer: Maybe better to link to the contact sales form page? Less spam, better data collection etc.

Author: ```suggestion
Yes. If you're interested in a BAA for HIPAA compliance, please [contact us with information about your requirements](/contact-sales).  
```

## How to set PostHog up for HIPAA compliance

> We strongly recommend teams which need HIPAA compliance use PostHog Cloud under a BAA, rather than the legacy self-hosted product or Hobby versions. 

The best way to use PostHog in a way which is HIPAA compliant is with a BAA covering a PostHog Cloud instance. This ensures that PostHog remains scalable and that you have access to all premium features and security enhancements. 

> However, it is possible to host PostHog yourself using either our open-source hobby deployment. It is important to note that the hobby deployment is only suitable for smaller event volumes and provided without guarantee. It also lacks many of the advanced features available under PostHog Cloud. 

Reviewer: ```suggestion
However, it is possible to host PostHog yourself using our open-source hobby deployment. It is important to note that the hobby deployment is **only suitable for smaller event volumes** and provided without guarantee. It also lacks many of the advanced features available under PostHog Cloud. 
```

> However, it is possible to host PostHog yourself using either our legacy self-hosted product, or our open-source hobby deployment. It is important to note that the legacy product is no longer supported, and that the hobby deployment is only suitable for smaller event volumes and provided without guarantee. Both versions lack many of the advanced features available under PostHog Cloud. 

Reviewer: I don't think we should mention the "legacy" product here. Afaik, you can't get a license anyway so is it event possible?

Author: ```suggestion
However, it is possible to host PostHog yourself using either our open-source hobby deployment. It is important to note that the hobby deployment is only suitable for smaller event volumes and provided without guarantee. It also lacks many of the advanced features available under PostHog Cloud. 
```
